CHANGES.md

```diff
@@ -223,6 +223,10 @@ OpenSSL 3.2
 
    *Hugo Landau*
 
+ * Fix enable the CCM mode of KTLS.
```

Comment:
This entry needs to describe any compatibility breaks with older kernels when CCM is enabled, e.g. something like "Note that linux kernel versions a.b.c - x.y.z have a known bug in CCM processing. KTLS with CCM ciphersuites should not be used on those linux kernels" (Obviously insert the correct broken kernel versions)